src/Controller/PrivacyController.php

```diff
@@ -32,13 +32,13 @@ public function privacyAction(Request $request) : Response
                     new Paragraph('We operate a permission-based email communication system, which allows the collection of personal information to send emails to people who have consented to receive them. Each email contains the option to unsubscribe. We may also use this data for the legitimate interest of business analysis and planning.'),
                     new Paragraph('Cookies allow us to make the website work more smoothly for users. For users who are logged in, the cookies allow them to move between pages and remain logged in. We use analytics cookies to allow a better understanding, at an anonymous aggregated level, of how users use the site, and broad categories of users, including their location. Local storage is an area of your browser used by websites to store information on your computer rather than making additional web requests, and can speed up browsing.'),
-                    new Paragraph('Visitors who do not wish to have cookies placed on their computers should set their browsers to refuse cookies before using eLife’s websites, with the drawback that certain features of our websites may not function properly without the aid of cookies. You can modify your browser settings to notify you each time a cookie is sent to it and you can decide whether or not to accept. Alternatively, you can set your browser to refuse all cookies. Cookies that have already been set can be deleted at any time using your browser settings. The independent <a href="https://www.aboutcookies.org/">AboutCookies.org</a> website contains comprehensive information on how to modify your browser settings. You can conceal your true IP address by using a virtual private network (VPN).')
+                    new Paragraph('Visitors who do not wish to have cookies placed on their computers should set their browsers to refuse cookies before using eLife’s websites, with the drawback that certain features of our websites may not function properly without the aid of cookies. You can modify your browser settings to notify you each time a cookie is sent to it and you can decide whether or not to accept. Alternatively, you can set your browser to refuse all cookies. Cookies that have already been set can be deleted at any time using your browser settings. The independent <a href="https://www.aboutcookies.org/">AboutCookies.org</a> website contains comprehensive information on how to modify your browser settings. You can conceal your true IP address by using a virtual private network (VPN). You can opt out of interaction tracking by enabling Do Not Track (DNT) in your browser.')
```

Comment:
`"You can opt out of interaction tracking by enabling Do Not Track (DNT) in your browser."` Is this true? DNT is only a signal sent to the server, it's up to the server to honour it: there's nothing inherently non-tracking in setting DNT. This is something that I've raised before, but I don't believe that at the moment we respond to client DNT signals. Does HotJar?

Reply:
@Maelplaine Can we get clarity on this?

Reply:
@discodavey Sorry I do not have the answer to that question. @diversemix do you know how HotJar works and can you answer @davidcmoulton 's question? We are updating the text of the privacy policy.

Reply:
Hotjar will respect Do Not Track (https://www.hotjar.com/legal/compliance/opt-out). The wording here specifically states that to opt out of `interaction tracking` you can enable DNT so we don't need to do that for other parts of journal as they collect data in aggregate for things like GA or New Relic. We can also use Hotjar on journal should we need to under this change. 

Having spoken to @davidcmoulton about this, we're happy to consider that any work/feature added to journal that is considered interaction tracking in the future will also include work to respect DNT. This is something we should be considering anyway and now gives us a privacy policy to ensure we do that. This all works in favour of privacy which is the right balance in my opinion.

Reply:
For the avoidance of doubt, this means we can go live with this change and we must all remember that any work that involves anything like interaction tracking needs to have privacy thought about and DNT respected in the future.

Reply:
Thanks for looking into this @BlueReZZ